importScripts/sql/cleaning/errorFields.sql

```diff
@@ -0,0 +1,148 @@
+# Result 262
+SELECT COUNT(*) from ObjectsExport WHERE  LENGTH(Recommendations) > 255;
+# Result 1
+
+
+# Could not add intrinsic  to ca_objects: surveyed By must be no more than 30 characters long
+SELECT max(length(ConditionSurveyedBy)) from ObjectsExport;
+# Was 30 updated to 255
+# Result 33
+SELECT count(*) from ObjectsExport WHERE LENGTH(ConditionSurveyedBy) > 30;
+# Result 1
+
+#Collation (Physical Description) must be no more than 100 characters long
+SELECT MAX(LENGTH(Collation)) from LibraryExport;
+# Was 100 updated to 255
+# Result 238
+SELECT COUNT(*) from LibraryExport WHERE LENGTH(Collation) > 100;
+# Result 17
+
+# Copy Number must be no more than 5 characters long
+SELECT max(LENGTH(CopyNo)) from LibraryExport;
+# Was 5 updated to 64
+# result 17
+SELECT count(*) FROM LibraryExport WHERE length(CopyNo) > 5;
+# Result 142
+
+SELECT MAX(LENGTH(Review)) from LibraryExport;
+# Result: 333
+# Was 255 updated to 1023
+SELECT count(*) from LibraryExport WHERE LENGTH(Review) > 255;
+# Result : 3
+# Invalid Review; values were Review = Newsletter RWAHS, Vol. 12, no. 3, April 1973 p. 8/R.P.W.also review newsletter Vol 13 no. 10 Nov/Dec, 1974 p. 6, review Vol. 3 RWAHS newsletter Vol 8, no. 4, p. 5/E.S.W. review Vol. 17  History West Vol. 47, no. 2 March 2008 p.5/Anne Rogers
+# Invalid Review; values were Review = of 'Per Ardua'; ''Moonda''; ''The Seabrooks and the Robinsons''; ''Old Brookton Flour Mill''; draft of ''Pompiya and Kouok''; ''Pompiya and Kouok'', ''Memoirs of Wade, William''; Bates, Daisy; Sir Rason, Cornthwaite Hector; Mattinson, J. A.; Yule, Thomas; interview with Bateman, Topsy  (Accession no. 1996.54a-j)
+# Invalid Review; values were Review = RWAHS newsletter, Vol.27, no.2, March 1988, p.8Accession number HS PR/795
+
+# ISBN/ISSN must be no more than 13 characters long
+SELECT max(length(ISBN)) FROM LibraryExport;
+# Result 16
+# Was 13 updated to 24
+SELECT count(*) from LibraryExport WHERE LENGTH(ISBN) > 13;
+# Result 1
+
+
+# Size must be no more than 20 characters long
+SELECT MAX(LENGTH(Size)) FROM LibraryExport ;
+# Result 39
+# Was 20 now 64
+SELECT COUNT(*) from LibraryExport WHERE LENGTH(Size) > 20;
+# Result 226
+
+
+# update library export mapping
+/*
+Could not set preferred label for entity Access to these Private Papers is restricted. For further information please contact the RWAHS Library. Seabrook, Jack; ''Undercliffe: Home and Hospital''; Miles, Norma; ''John Seabrook of Brookton House''; ''Memoirs' an autobiography by Leggoe, John; ''Woodich''; Bussell, Bessie, ''So Harsh their Land'', Monger, Peter; review of 'Per Ardua'; ''Moonda''; ''The Seabrooks and the Robinsons''; ''Old Brookton Flour Mill''; draft of ''Pompiya and Kouok''; ''Pompiya and Kouok'', ''Memoirs of Wade, William''; Bates, Daisy; Sir Rason, Cornthwaite Hector; Mattinson, J. A.; Yule, Thomas; interview with Bateman, Topsy (Accession no. 1996.54a-j) Donor: A. Lendall: Surname/organisation must not be more than 100 characters long
```

Comment:
These donors were fixed using the Donors worksheet in the library import mapping